src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,84 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    # Linking our PDBs to Git is really only interesting for PE / PDB that are 
+    # being shipped to customers.  For local developement the original PDB 
```

Comment:
I think you need to consider the cost of this tool.  PdbGit is not a fast tool, instead it's comparitevyl slow with other tools used during build.  The performance seems to scale with the size of the binary involved.  This means adding this to our unit test runs will add quite a bit of time to our unit tests runs.  

To give a sense of how much consider that we are linking ~60 PDBs and it takes 15 minutes to do so.  If we implement source linking perfectly on our source tree that will be ~200 PDBs and hence easily adding about 1 hour to our unit test time.  

By perfectly I mean that we link a PDB exactly once.  Given our build setup PDBs will appear multiple times in the output tree (due to build output copying).  After the first link we need to copy the PDB vs. linking again as copying is faster.  That's certainly doable but takes a bit more effort. 

I'm currently digging into the performance a bit to see if there are any wins we could get.  At this time though I don't think it's viable for unit test legs.